karma.conf.js

```diff
@@ -0,0 +1,43 @@
+var webpackConfig = require("./webpack.config");
+Object.assign(webpackConfig, {
+    debug: true,
+    devtool: "inline-source-map"
+});
+
+webpackConfig.externals.push("react/lib/ExecutionEnvironment");
+webpackConfig.externals.push("react/lib/ReactContext");
+webpackConfig.externals.push("react/addons");
+webpackConfig.externals.push("jsdom");
+
+module.exports = function(config) {
+    config.set({
+        basePath: "",
+        frameworks: [ "jasmine" ],
+        files: [
+            { pattern: "src/**/*.ts", watched: false, included: false, served: false },
+            { pattern: "tests/**/*.ts", watched: false, included: false, served: false },
```

Comment:
If all 3 flags are false afaik it is effectively the same as not adding the line in the config.